### PR TITLE
First export plugin code

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: openslides_export/views.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    def get(self, request, *args, **kwargs):
+        return self.render_to_response()
+
+    def get_filename(self):
+        """
+        Return the filename for the output file (without suffix).
+        """
+        if self.get_object():
+            motion = Motion.objects.get(pk=self.get_object().pk)
```
Comment (normanjaeckel) [inline openslides_export/views.py]: This line is redundant for get_object() already contains the motion.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from setuptools import find_packages, setup
+
+package_name = 'openslides-export'
+module_name = 'openslides_export'
+
+# The following commented unique string is used to detect this import.
+module = __import__(module_name)  # EuC5Kee6ohp7aesae3yai3fe3soo2uzoshohk2uj
```
Comment (normanjaeckel) [inline setup.py]: The string should be unique and not the same as in other plugins to be sure the the import restriction actually works well.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+language: python
```
Comment (normanjaeckel) [inline .travis.yml]: Travis ahs to be activated in GitHub settings.

File: openslides_export/views.py (hunk 1 above)
Comment (emanuelschuetze) [inline openslides_export/views.py]: done

File: setup.py (hunk 2 above)
Comment (emanuelschuetze) [inline setup.py]: done.

File: .travis.yml (hunk 3 above)
Comment (emanuelschuetze) [inline .travis.yml]: ok, I'll activate it after merge.

**Pull request merged**